addons/l10n_ma/data/template/account.account-ma.csv

```diff
@@ -1,635 +1,635 @@
-"pcg_75861","75861","Donations","income_other","False","","Dons"
-"pcg_75862","75862","Liberalities","income_other","False","","Libéralités"
-"pcg_75863","75863","Prizes","income_other","False","","Lots"
-"pcg_7588","7588","Other Non-Current Income from Previous Years","income_other","False","","Autres produits non courants des exercices antérieurs"
-"pcg_75911","75911","Non-Current Reversals of Exceptional Depreciation of Non-Current Assets in Non Values","income_other","False","","Reprises non courantes sur amortissements exceptionnels de l'immobilisation en non valeurs"
-"pcg_75912","75912","Non-Current Reversals of Exceptional Depreciation of Intangible Non-Current Assets","income_other","False","","Reprises non courantes sur amortissements exceptionnels des immobilisations incorporelles"
-"pcg_75913","75913","Non-Current Reversals of Exceptional Depreciation of Tangible Non-Current Assets","income_other","False","","Reprises non courantes sur amortissements exceptionnels des immobilisations corporelles"
-"pcg_75941","75941","Reversals of Accelerated Depreciation","income_other","False","","Reprises sur amortissements dérogatoires"
-"pcg_75942","75942","Reversals of Capital Gains Pending Taxation","income_other","False","","Reprises sur plus-values en instance d'imposition"
-"pcg_75944","75944","Reversals of Allocations for Investments","income_other","False","","Reprises sur provisions pour investissements"
-"pcg_75945","75945","Reversals of Allocations for Replenishment of Deposits","income_other","False","","Reprises sur provisions pour reconstitution de gisements"
-"pcg_75946","75946","Reversals of Allocations for the Acquisition and Construction of Housing","income_other","False","","Reprises sur provisions pour acquisition et construction de logements"
-"pcg_75955","75955","Reversals of Provisions for Long-Term Risks and Expenses","income_other","False","","Reprises sur provisions pour risques et charges durables"
-"pcg_75957","75957","Reversals of Provisions for Temporary Risks and Expenses","income_other","False","","Reprises sur provisions pour risques et charges momentanés"
-"pcg_75962","75962","Non-Current Reversals of Provisions for Depreciation of Non-Current Assets","income_other","False","","Reprises non courantes sur provisions pour dépréciation de l'actif immobilisé"
-"pcg_75963","75963","Non-Current Reversals of Provisions for Depreciation of Current Assets","income_other","False","","Reprises non courantes sur provisions pour dépréciation de l'actif circulant"
-"pcg_7597","7597","Non-Current Expense Transfers","income_other","False","","Transferts de charges non courantes"
-"pcg_7598","7598","Non-Current Reversals from Previous Years","income_other","False","","Reprises non courantes des exercices antérieurs"
+"id","code","name","account_type","reconcile","non_trade","name@fr","asset_model_ids","description","tax_ids"
```

Comment:
Since you added the description you should also add a column description@fr for the translation of the description :smile: